ioctl/newcmd/config/config.go

```diff
@@ -128,6 +131,78 @@ func (c *info) reset() error {
 	return nil
 }
 
+// set sets config variable
+func (c *info) set(args []string, insecure bool) (string, error) {
+	switch args[0] {
+	case "endpoint":
+		if !isValidEndpoint(args[1]) {
+			return "", errors.New(fmt.Sprintf("endpoint %s is not valid", args[1]))
+		}
+		c.readConfig.Endpoint = args[1]
+		c.readConfig.SecureConnect = !insecure
+	case "analyserEndpoint":
+		c.readConfig.AnalyserEndpoint = args[1]
+	case "wallet":
+		c.readConfig.Wallet = args[1]
+	case "explorer":
+		lowArg := strings.ToLower(args[1])
+		switch {
+		case isValidExplorer(lowArg):
+			c.readConfig.Explorer = lowArg
+		case args[1] == "custom":
+			fmt.Println(`Please enter a custom link below:("Example: iotexscan.io/action/")`)
+			var link string
+			_, err := fmt.Scanln(&link)
```

Comment:
These should be put into `client` (refer to `AskToConfirm`).  call `client.func` here.   we can mock the new function in test, so the left code can be run.

Reply:
Added and passed the client into the function, let me know if that's the solution you had in mind.